### ISSUE TITLE
Deleted a pdf file on my android phone. 

### ISSUE THREAD

**Issue by bapjg** [nextcloud/android#7494]
### Steps to reproduce
I chose to delete the file on the handset and on the server. Immediately it crashed (report enclosed below). On the phone, the file had been deleted. I synchronised the directory and noticed that they was still no pdf file, so I presume it had been deleted on the server too. Therefore, it seems that the crash occurred after the entire delete process (handset and server) 
2. 
3. 
************ CAUSE OF ERROR ************

android.app.RemoteServiceException: Context.startForegroundService() did not then call Service.startForeground(): ServiceRecord{15ea877 u0 com.nextcloud.client/.media.PlayerService}
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2313)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:213)
	at android.app.ActivityThread.main(ActivityThread.java:8178)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:513)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1101)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30140151
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: HONOR
Device: HWBKL
Model: BKL-L09
Id: HUAWEIBKL-L09S
Product: BKL-L09

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: 10.0.0.177C432

### Expected behaviour
- Tell us what should happen

### Actual behaviour
- Tell us what happens

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by axheli** [comment]
Media Tab loading issues #7245

**Comment by axheli** [comment]
#7461

**Comment by tobiasKaminsky** [comment]
This is fixed via 3.14.1 RC2, please check via play store beta program.
